common/determined_common/storage/shared.py

```diff
@@ -56,3 +57,22 @@ def get_mount_config(self) -> Dict[str, Any]:
             "ReadOnly": False,
             "BindOptions": {"Propagation": self.propagation},
         }
+
+    @contextlib.contextmanager
+    def restore_path(self, metadata: StorageMetadata) -> Iterator[str]:
+        """
+        Prepare a local directory exposing the checkpoint. Do some simple checks to make sure the
+        configuration seems reasonable.
+        """
+        storage_dir = os.path.join(self._base_path, metadata.storage_id)
+        check_true(
+            os.path.exists(storage_dir),
+            "Storage directory does not exist: {}. Please verify "
+            "that you are using the correct configuration value for "
+            "checkpoint_storage.host_path and "
+            "tensorboard_storage.host_path.".format(storage_dir),
```

Comment:
blocking: The `tensorboard_storage` key is no longer in use so we should update this log message to reflect that

Reply:
done